Templates/AppSource App/.github/workflows/AddExistingAppOrTestApp.yaml

```diff
@@ -19,8 +19,18 @@ defaults:
     shell: PowerShell
 
 jobs:
+  initialization :
+    runs-on: windows-latest
+    outputs:
+      telemetryScope: ${{ steps.init.outputs.telemetryScope }}
+    steps:
+      - id: init
+        uses: microsoft/AL-Go/InitWorkflow@main
+        with: 
+          eventId : "DO0090"
```

Comment:
Would it be better to use event Names (e.g. action:AddExistingAppOrTestApp) - and then find the EventId from a table and add the message to the string?